src/app/component/CheckoutOrderSummary/CheckoutOrderSummary.component.js

```diff
@@ -0,0 +1,135 @@
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isItemListOpen: false,
+        }
+    }
+
+    /**
+     * Render price line
+     */
+    renderPriceLine(price, name, mods) {
+        return (
+            <li block="CheckoutOrderSummary" elem="SummaryItem" mods={ mods }>
+                <strong block="CheckoutOrderSummary" elem="Text" mods={ { align: 'left' } }>{ name }</strong>
+                <strong block="CheckoutOrderSummary" elem="Text" mods={ { align: 'right' } }>
+                    {/* TODO: Use value from configuration file */ }
+                    $
+                    <TextPlaceholder content={ price }/>
+                </strong>
+            </li>
+        );
+    }
+
+    /**
+     * Render order summury cart item
+     * @param key
+     * @param item
+     * @returns {*}
+     */
+    renderItem(key, item) {
+        return (
+            <li key={ key } block="CheckoutOrderSummary" elem="CartItem">
+                <div
+                    block="CheckoutOrderSummary"
+                    elem="Thumbnail"
+                    aria-label="Cart Thumbnail"
+                >
+                    <Image src={ `/media/catalog/product${ item.thumbnail }` } alt="Cart Thumbnail"/>
+                </div>
+
+                <div block="CheckoutOrderSummary" elem="Title">
+                    { item.manufacturer && <span>{ item.manufacturer }</span> }
+                    <p><TextPlaceholder content={ item.name }/></p>
+                    <p><TextPlaceholder content={ `Qty: ${ item.quantity }` }/></p>
+                </div>
+
+                <div
+                    block="CheckoutOrderSummary"
+                    elem="Details"
+                >
+                    <div block="CheckoutOrderSummary" elem="Price">
+                        <ProductPrice price={ item.price } mods={ { type: 'regular' } }/>
+                    </div>
+                </div>
+            </li>
+        );
+    }
+
+    /**
+     * Render checkout order summary block
+     * @returns {*}
+     */
+    render() {
+        const { isItemListOpen } = this.state; //@TODO implement open close
+        const {
+            totals: { grandTotalPrice },
+            products,
+            shippingMethod,
+        } = this.props;
+
+        // calculate grand totals including shipping price
+        const grandTotalWithShipping = (shippingMethod.price) ? parseFloat(grandTotalPrice) + parseFloat(shippingMethod.price) : grandTotalPrice;
+
+        return (
+            <div block="CheckoutOrderSummary" aria-label="Order Summary">
+                <div block="CheckoutOrderSummary" elem="OrderTotals">
+                    <h3>Order Summary</h3>
+                    <ul>
+                        { this.renderPriceLine(grandTotalPrice, 'Cart Subtotal') }
+                        { shippingMethod.price && this.renderPriceLine(String(shippingMethod.price), `Shipping (${ shippingMethod.title })`, { divider: true }) }
```

Comment:
Thanks for pointing out, otherwise I would have forgotten to file bug report. So basically there is issue with placeholder. it expects string which means you can't render numbers with placeholder:
` Failed prop type: Invalid prop `content` of type `number` supplied to `TextPlaceholder`, expected `string`.`

I casted them as temp solution to get rid of warnings. filed bug report #93